django_ses/utils.py

```diff
@@ -187,3 +188,13 @@ def verify_bounce_message(msg):
     """
     verifier = BounceMessageVerifier(msg)
     return verifier.is_verified()
+
+
+def to_bytes(bytes_or_str):
+    # From: https://stackoverflow.com/a/46037362
+    if isinstance(bytes_or_str, str):
+        value = bytes_or_str.encode() # uses 'utf-8' for encoding
```

Comment:
Why not just pass `'utf-8'` here then?  (shorter and no chance for nitpicking that the comment should have two spaces in front ;D)

Reply:
Could also return it there already, making the `else:` unnecessary`.
Also the comment there (with the `return`) is unnecessary.

Reply:
Good points, such is the mishap of pasting SO code sometimes.